### ISSUE TITLE
xmipp particle picker import single pos file

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#602]
In xmipp manual picker I have selected File-Export coordinates and saved them to _lol.pos_ file (it has coords for all mics). However I can't import this single file back to xmipp picker since it suggests me to import only for 1-st selected micrograph:

![de9](https://cloud.githubusercontent.com/assets/6952870/19196711/dd940300-8cb6-11e6-9ed4-b51781f41457.png)

**Comment by azazellochg** [comment]
Alright, first if was not importing anything at all. I kind of made it work by explicitly selecting format Xmipp24. By the way would be nice to know the differences in pos file format Xmipp24/30/31 and why Auto is not working.. Anyway, the problem persists: all particles from single pos file are imported into a single mic.
Also, scipion import coordinates protocol does not import a single pos file with coords for all mics..

**Comment by delarosatrevin** [comment]
Yes, I think the .pos extension was confusing as Xmipp2.4 (although
probably it is written there as Xmipp 3.x). I agree that we should add an
entry in the wiki with the different among Xmipp coordinate formats. We
should modify the Java particle picker to correctly import coordinates from
a single file....and also in Scipion to handle this case properly.

**Comment by azazellochg** [comment]
Another related issue is when importing multiple pos files in _scipion import coordinates_ protocol, the header produced by xmipp picker export function is not valid:
```
# XMIPP_STAR_1 * 
# 
data_mic_img_0002
loop_
 _xcoor
 _ycoor
 _cost
```
The only header that worked for me is the one without mic name:
```
# XMIPP_STAR_1 * 
# 
data_particles
loop_
 _xcoor
 _ycoor
 _cost
```
